assets/helpers/report-card.js

```diff
@@ -0,0 +1,24 @@
+import ReportCard from './card-container.js'
+
+class MyReport extends ReportCard {
+	constructor(id, category, hazard, location, date, photos, comment) {
+		super(id, category, hazard, location, date, photos, comment)
+	}
+
+	reportContent() {
+		let div = document.createElement('div')
+		div.setAttribute('id', `reportCard${this.id}`)
+		div.innerHTML = `
+        <h2 class="hazardTitle">${this.hazard}</h2>
+		<p class="hazardLocation">${this.location}</p>
+		<p class="hazardDate">${this.date}</p>
+		<p class="hazardTime">${this.date}</p>
```

Comment:
same for the rest of the classes here.

Reply:
done https://github.com/whamester/WMDD-4885-Project/pull/24/commits/d459103d473bf92e8c2ae9ecf4491568af762f6a